runtime/bindings/python/tests/hal_test.py

```diff
@@ -303,13 +303,14 @@ def testDifferentSizeQueueCopy(self):
             allocation_size=13,
         )
         sem = self.device.create_semaphore(0)
-        with self.assertRaisesRegex(ValueError, "length must match"):
-            self.device.queue_copy(
-                source_buffer,
-                target_buffer,
-                wait_semaphores=iree.runtime.HalFence.create_at(sem, 0),
-                signal_semaphores=iree.runtime.HalFence.create_at(sem, 1),
-            )
+        self.device.queue_copy(
```

Comment:
Let's change this test name to 'testIncompatibleSizeQueueCopy` and still test the exception case by having a source size greater than target.